### PR TITLE
Search

### PR DESCRIPTION
**Pull request by amdaddeo**
There is a slight problem with the hibernate but doesn't effect any of the results. If you know about hibernate please take a look!

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,29 @@
```
Comment (amshali) [inline .project:0]: Please add this file to your .gitignore.
Remove it.
commit again and push.

File: app/controllers/UserSearch.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import play.exceptions.TemplateNotFoundException;
+import play.modules.elasticsearch.search.SearchResults;
+import play.mvc.Before;
+import controllers.*;
+import controllers.elasticsearch.ElasticSearchController;
+import controllers.elasticsearch.ElasticSearchController.ObjectType;
+import models.*;
+
+@ElasticSearchController.For(User.class)
+public class UserSearch extends Application {
```
Comment (amshali) [inline app/controllers/UserSearch.java:18]: Must extend OBController

File: app/controllers/Users.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import org.elasticsearch.index.query.QueryBuilders;
+
+import models.Status;
+import models.User;
+import play.modules.elasticsearch.ElasticSearch;
+import play.modules.elasticsearch.search.SearchResults;
+import play.mvc.Controller;
+
+
+public class Users extends Controller {
```
Comment (amshali) [inline app/controllers/Users.java:12]: What is this? This should already be there... strange

File: conf/routes [hunk 4]
```diff
@@ -48,6 +52,7 @@ GET     /users/{<[0-9]+>ownerId}/photos/?       Photos.photos
 # Checkin
 GET     /checkin                                Checkin.checkin
 GET     /checkin/at                             Checkin.at #what is this?  maybe should be post
+>>>>>>> b32ecb13076c837720c34a473a5684403a755a92
```
Comment (amshali) [inline conf/routes:55]: What is this line? Have you tested your code?

File: .gitignore [hunk 5]
```diff
@@ -8,6 +8,7 @@
```
Comment (amshali) [inline .gitignore:7]: Please please get the latest update from main repo. This is already there.

File: app/controllers/UserSearch.java [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            if (getControllerClass() == ElasticSearchController.class) {
+                forbidden();
+            }
+            render("UserSearch/index.html");
+        }
+        
+        /**
+         * Search.
+         *
+         * @param page the page
```
Comment (laurenyew) [inline app/controllers/UserSearch.java:33]: What do you mean that you can search the page? Do you mean searching for a page that has the fields you're looking for? If I search for a userName, can I get the user's account page and edit it?

**Comment by laurenyew** [comment]
Is there a reason that you put the searchable box in the header right next to the logo? 
For readability purposes, can it be more to the right? 
Also, is there a way the box can be labeled, so a new user can more easily tell what it is? 
(For example, Facebook has the Search box say "Search". You can do this by 
<input type = "text" name = whateveryounamedit value = "Search" />
Thanks!

**Pull request closed without merging**